scripts/upload_website_demos.py

```diff
@@ -93,13 +95,15 @@ def upload_demo_to_space(
 if __name__ == "__main__":
     parser = argparse.ArgumentParser()
     parser.add_argument("--WHEEL_URL", type=str, help="aws link to gradio wheel")
+    parser.add_argument("--CLIENT_URL", type=str, help="gradio version")
     parser.add_argument("--AUTH_TOKEN", type=str, help="huggingface auth token")
     parser.add_argument("--GRADIO_VERSION", type=str, help="gradio version")
     args = parser.parse_args()
     gradio_wheel_url = args.WHEEL_URL + f"gradio-{args.GRADIO_VERSION}-py3-none-any.whl"
+    
     if args.AUTH_TOKEN is not None:
         hello_world_version = str(huggingface_hub.space_info("gradio/hello_world").cardData["sdk_version"])
         for demo in demos:
             if hello_world_version != args.GRADIO_VERSION:
                 upload_demo_to_space(demo_name=demo, space_id="gradio/" + demo, hf_token=args.AUTH_TOKEN, gradio_version=args.GRADIO_VERSION)
-            upload_demo_to_space(demo_name=demo, space_id="gradio/" + demo + "_main", hf_token=args.AUTH_TOKEN, gradio_version=args.GRADIO_VERSION, gradio_wheel_url=gradio_wheel_url)
+            upload_demo_to_space(demo_name=demo, space_id="gradio/" + demo + "_main", hf_token=args.AUTH_TOKEN, gradio_version=args.GRADIO_VERSION, gradio_wheel_url=gradio_wheel_url, gradio_client_url=args.CLIENT_URL)
```

Comment:
We modify args.WHEEL_URL by adding "gradio-{args.GRADIO_VERSION}-py3-none-any.whl" to it. Do we need to do the same thing for the `gradio_client`?

Reply:
Nope, the gradio_client is just installed from github, so there's no actual wheel file. it's just something like 
`
gradio-client @ git+https://github.com/gradio-app/gradio@c0d25fc1df60987793471b4745ab453bea92dcfe#subdirectory=client/python
`
that's also what we do for the pr demos

Reply:
Ok great! Thanks for clarifying